includes/integrations/woocommerce-memberships/class-member-chi-woocommerce-memberships-integration.php

```diff
@@ -111,6 +116,10 @@ public function membership_status_changed( $user_membership, $old_status, $new_s
 			'old_status' => $old_status,
 		);
 
+		if ( 'expired' === $old_status ) {
```

Comment:
Wouldn't this set the `date_expired` to `time()` when a membership is going _from_ `expired` to something else?

Maybe you meant to check if `expired` is `$new_status` instead? Also possible I'm missing something 😄